# HG changeset patch
# User Rik <rik@octave.org>
# Date 1452024301 28800
#      Tue Jan 05 12:05:01 2016 -0800
# Branch stable
# Node ID 8fdf1f991a32ae184561187d113e104d17897f02
# Parent  c27b6503e5d2bfbbdc7d58387857eb143c7b8c22
maint: Add @SizeTester files to build system.

* test/classes/module.mk: Add @SizeTester files to build system.

diff --git a/test/classes/module.mk b/test/classes/module.mk
--- a/test/classes/module.mk
+++ b/test/classes/module.mk
@@ -102,23 +102,29 @@ class_CPrecedenceTester1_FCN_FILES = \
 class_CPrecedenceTester2_FCN_FILES = \
   classes/@CPrecedenceTester2/CPrecedenceTester2.m \
   classes/@CPrecedenceTester2/tattack.m
 
 class_CPrecedenceTester3_FCN_FILES = \
   classes/@CPrecedenceTester3/CPrecedenceTester3.m \
   classes/@CPrecedenceTester3/tattack.m
 
+class_SizeTester_FCN_FILES = \
+  classes/@SizeTester/SizeTester.m \
+  classes/@SizeTester/numel.m \
+  classes/@SizeTester/size.m
+
 classes_FCN_FILES = \
   $(class_Blork_FCN_FILES) \
   $(class_Cork_FCN_FILES) \
   $(class_Dork_FCN_FILES) \
   $(class_Gork_FCN_FILES) \
   $(class_Pork_FCN_FILES) \
   $(class_Sneetch_FCN_FILES) \
   $(class_Snork_FCN_FILES) \
   $(class_Spork_FCN_FILES) \
   $(class_CPrecedenceTester1_FCN_FILES) \
   $(class_CPrecedenceTester2_FCN_FILES) \
   $(class_CPrecedenceTester3_FCN_FILES) \
+  $(class_SizeTester_FCN_FILES) \
   classes/classes.tst
 
 FCN_FILES += $(classes_FCN_FILES)
